src/main/kotlin/io/github/mojira/arisa/infrastructure/config/Arisa.kt

```diff
@@ -85,8 +93,9 @@ object Arisa : ConfigSpec() {
 
         object FutureVersion : ModuleConfigSpec() {
             val futureVersionMessage by optional(
-                "Please *do not* mark _unreleased versions_ as affected.\r\nYou don't have access to them yet."
-            )
+                "{panel:borderColor=orange}(!) Please do not mark _Unreleased Versions_ as affected. You don't have access to them yet.{panel}"
```

Comment:
`+`

Reply:
How's that? Or did I not understand what you meant?